istio-proxy/fetch.sh

```diff
@@ -13,18 +13,39 @@ function check_envs() {
   CACHE_DIR=${PROXY_FETCH_DIR}/bazel
 }
 
-function set_default_envs() {
-  if [ -z "${PROXY_GIT_REPO}" ]; then
-    PROXY_GIT_REPO=https://github.com/istio/proxy
+function check_git_hash() {
+  local name "$1"
+  local commit_hash_name "$2"
+
+  eval commit_hash=\$$commit_hash_name
+  if [ -z "${commit_hash}" ]; then
+    echo "ERROR: ${name} was set but missing ${commit_hash_name}";
+    exit 1;
   fi
+}
 
-  if [ -z "${OPENSSL_GIT_BRANCH}" ]; then
-    OPENSSL_GIT_BRANCH=maistra-1.1
-  fi
+function extract_dependency() {
+  local name="$1"
+  local repo="$2"
+  local commit="$3"
 
-  if [ -z "${PROXY_GIT_BRANCH}" ]; then
-    PROXY_GIT_BRANCH=release-1.3
-  fi
+  wget "${repo}/archive/${commit}.tar.gz"
+  tar xvf "${commit}.tar.gz"
+  rm "${commit}.tar.gz"
+  mv "${name}-${commit}" "${name}"
+}
+
+function set_default_envs() {
+  PROXY_GIT_REPO=https://github.com/maistra/proxy
+  ISTIO_PROXY_OPENSSL_GIT_REPO=https://github.com/maistra/istio-proxy-openssl
+  ENVOY_OPENSSL_GIT_REPO=https://github.com/brian-avery/envoy-openssl
```

Comment:
This should be moved to under maistra github organisation?

Reply:
Yep, at the time these PRs were open, that code wasn't merged. Will fix it now.